examples/load_balancer/load_balancer.c

```diff
@@ -600,6 +674,10 @@ main(int argc, char *argv[]) {
         int arg_offset;
         const char *progname = argv[0];
 
+        time_t t;
+	/* Intializes RANDOM number generator */
+	srand((unsigned) time(&t));
```

Comment:
Check indentation

Reply:
What is the issue with the indentation here?

Reply:
looks like tabs not spaces. @catherinemeadows -- do you know if we have a working linter script that checks for this kind of formatting issue in ONVM? I thought our GitHub actions would do it automatically, but it seems not.

Reply:
Yes, in `/openNetVM/style`, there is documentation about manually running our linter scripts.